Assets/Scripts/Vision/RendererEffect.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Anivision.Core;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace Anivision.Vision
+{
+    // Parent class for effects that modify the render pipeline
+    public abstract class RendererEffect : MonoBehaviour
+    {
+        public VisionEffect Effect { get; } // The vision effect that corresponds to this renderer effect
```

Comment:
maybe you want to make this an abstract, so classes inheriting from this are forced to use override and implement a getter for this.

Reply:
Yeah, I tried that initially, but variables set as override / abstract were not visible in the inspector, so I couldn't make it into a dropdown.

Reply:
Why does it need to be exposed in the inspector? Shouldn't all classes that inherit from this have a set VisionEffect that shouldn't be modified through the inspector? For example, the Colorblind script should always return a VisionEffect of Colorblind, no?

Reply:
Yes, that's true for the most part. BlitEffect.cs is a more general purpose effect that can be modified in the inspector though.

Reply:
oh so `BlitEffect.cs` allows easy customization of a blit effect if people don't want to write their own scripts? that's cool. ok, got it.

Reply:
Yes, that was the goal at least